### ISSUE TITLE
Provide ability to set the auth guard via config like nova

### ISSUE THREAD

**Issue by leeovery** [spatie/mailcoach-support#11]
Hey

Thanks for building something so polished - every piece from the UI to the codebase is beautiful - amazing work!!!!

One issue I've stumbled into is that the Gate check is failing due to trying to use my default auth guard - which is not the guard I want to use, and as such is not even executing the gate callback (it's just failing and redirecting to the route configured in mailcoach.php config).

In short - can we add a config to configure the guard mailcoach uses to resolve the User, please? Nova took this route and it seemed to work pretty well. 

I'm using MC as a package inside an existing app FYI.

Thanks again!
Lee

**Comment by freekmurze** [comment]
I'll take care of this soon. Thanks for the suggestion.

**Comment by freekmurze** [comment]
This option is now available in `v1.2.0`.

Run `composer update` and add this to your `mailcoach` config file.

```
    /*
     *  This configuration option defines the authentication guard that will
     *  be used to protect your the Mailcoach UI. This option should match one
     *  of the authentication guards defined in the "auth" config file.
     */
    'guard' => env('MAILCOACH_GUARD', null),
```
